vi_minimal.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<p>Can you help me understand how this fixes the issue?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>